src/GlobalStyles.js

```diff
@@ -6,17 +6,10 @@ const styled = {createGlobalStyle};
 export default styled.createGlobalStyle`
   * {
     box-sizing: border-box;
+    font-family: Quicksand;
```

Comment:
```suggestion
    font-family: Quicksand, sans-serif;
```
It's always good to have a generic fallback for fonts.

From [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/font-family):
_"You should always include at least one generic family name in a font-family list, since there's no guarantee that any given font is available. This lets the browser select an acceptable fallback font when necessary."_